setup.py

```diff
@@ -1,19 +1,28 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+from tqdm._version import __version__
```

Comment:
I would prefer a system adapted from that : https://github.com/vispy/vispy/blob/master/setup.py#L51

(I can take care of it if you want)